linode/objkey/framework_resource.go

```diff
@@ -145,6 +145,7 @@ func (r *Resource) Update(
 ) {
 	tflog.Debug(ctx, "Update linode_object_storage_key")
 	var plan, state ResourceModel
+	tflog.Info(ctx, "here0")
```

Comment:
Do we still need this logging message? I think it can be removed.

Reply:
Thanks for the head up! I am going to clean them up.